### PR TITLE
Small edits to the manuscript for JOSS

### PR DESCRIPTION
**Pull request by dfm**
I've made a few small formatting changes to the paper as a final round of edits for https://github.com/openjournals/joss-reviews/issues/2582

### CODE REVIEW

**Review by steven-murray — APPROVED** [review]
Thanks @dfm. There's also another PR (#180) open that fixes some of the links/captions (actually, some of these fixes are there as well). Just waiting on a review from the team for that to be merged. Weirldy, compiling the paper with whedon from that branch doesn't seem to actually include the changes made there. I'll recompile on the Review issue once that has been merged...

**Comment by dfm** [comment]
@steven-murray: Thanks - I'd missed that, sorry! Let me know in the main thread once these are all merged.

**Pull request merged**
